htdocs/js/modules/direct_entry.js

```diff
@@ -63,7 +64,13 @@ $(document).ready(function() {
             value: nextPage
         }).appendTo("#test_form");
 
-        formEl.action = document.documentURI;
+        if (document.documentURI) {
+            documentURI=document.documentURI;
+        }
+        else {
+            documentURI= window.location.href;
```

Comment:
why not just always use window.location.href? It's standard and supported in all browsers. Is there any benefit to using document.documentURI on other browsers?

Reply:
@driusan Requested changes done.